### PR TITLE
Adds the Alien Organ Extractor

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Adds the Alien Organ Extractor, purchaseable for abductors for 2 tokens.
Advantages:
- It doesn't deal damage to the person to be operated on
- It's significantly faster; 3 seconds on other people, 1 second on yourself
- You can put organs in it manually
- Is EMP proof

Outstanding issues I'm fixing:
- [x] It doesn't move the organ out of my hand
- [x] It might not work, because I haven't tested with 2 clients yet

Also adds the price in tokens between brackets for the abductor console

## Why It's Good For The Game
Gives abductors a fancy new tool, to help with any mass abductions they might do

## Images of changes
image

image

## Changelog
:cl:
add: Adds the Alien Organ Extractor for 2 tokens
/:cl:

### CODE REVIEW

File: code/game/gamemodes/miniantags/abduction/machinery/console.dm [hunk 1]
```diff
@@ -65,12 +65,13 @@
 		dat += "Collected Samples : [points] <br>"
 		dat += "Gear Credits: [credits] <br>"
 		dat += "<b>Transfer data in exchange for supplies:</b><br>"
-		dat += "<a href='?src=[UID()];dispense=baton'>Advanced Baton</A><br>"
+		dat += "<a href='?src=[UID()];dispense=baton'>Advanced Baton (2)</A><br>"
 		dat += "<a href='?src=[UID()];dispense=helmet'>Agent Helmet</A><br>"
 		dat += "<a href='?src=[UID()];dispense=vest'>Agent Vest</A><br>"
 		dat += "<a href='?src=[UID()];dispense=silencer'>Radio Silencer</A><br>"
 		dat += "<a href='?src=[UID()];dispense=tool'>Science Tool</A><br>"
```
Comment (Burzah) [inline code/game/gamemodes/miniantags/abduction/machinery/console.dm:72]:
```suggestion
		dat += "<a href='?src=[UID()];dispense=baton'>Advanced Baton (2)</a><br>"
		dat += "<a href='?src=[UID()];dispense=helmet'>Agent Helmet</a><br>"
		dat += "<a href='?src=[UID()];dispense=vest'>Agent Vest</a><br>"
		dat += "<a href='?src=[UID()];dispense=silencer'>Radio Silencer</a><br>"
		dat += "<a href='?src=[UID()];dispense=tool'>Science Tool</a><br>"
```

File: code/game/gamemodes/miniantags/abduction/machinery/console.dm [hunk 2]
```diff
@@ -65,12 +65,13 @@
...
-		dat += "<a href='?src=[UID()];dispense=mind_device'>Mental Interface Device</A><br>"
+		dat += "<a href='?src=[UID()];dispense=mind_device'>Mental Interface Device (2)</A><br>"
+		dat += "<a href='?src=[UID()];dispense=organ_extractor'>Organ Extractor (2)</A><br>"
```
Comment (Burzah) [inline code/game/gamemodes/miniantags/abduction/machinery/console.dm:74]:
```suggestion
		dat += "<a href='?src=[UID()];dispense=mind_device'>Mental Interface Device (2)</a><br>"
		dat += "<a href='?src=[UID()];dispense=organ_extractor'>Organ Extractor (2)</a><br>"
```

**Review by Burzah — CHANGES_REQUESTED** [review]

**Comment by SchrodingersWolf** [comment]
A bit concerned about this, especially the fact you can manually put organs in them - Wouldn't that allow abductors to give themselves multiple implants easily? (I know it is possible even now, but it is a longer process compared to 1 second.)

**Comment by DGamerL** [comment]
> A bit concerned about this, especially the fact you can manually put organs in them - Wouldn't that allow abductors to give themselves multiple implants easily? (I know it is possible even now, but it is a longer process compared to 1 second.)

I rarely ever see abductors do this, and this really won't enable the strategy more. I see this as more of a faster sidegrade to doing normal surgery on humanoids

File: code/modules/surgery/organs/organ_extractor.dm [hunk 3]
```diff
@@ -154,3 +148,40 @@
...
+	img.plane = FLOAT_PLANE
+	overlays += img
+	storedorgan.pixel_x = organ_x
+	storedorgan.pixel_y = organ_y
+	overlays += "organ_extractor_2" //should look nicer for transparent stuff.
+
+/// Advanced abductor version. Is a lot faster with implanting into others
+/obj/item/organ_extractor/abductor
+	name = "alien organ extractor"
+	w_class = WEIGHT_CLASS_SMALL
```
Comment (Henri215) [inline code/modules/surgery/organs/organ_extractor.dm:173]:
```suggestion
```
That one too

**Review by Henri215 — APPROVED** [review]
Lewcs good to me

**Review by S34NW — APPROVED** [review]

**Pull request merged**